cli/auth_test.go

```diff
@@ -29,6 +33,25 @@ func TestPrintAccessTokenAction(t *testing.T) {
 	test.That(t, out.messages[0], test.ShouldContainSubstring, testToken)
 }
 
+func TestAPIKeyCreateAction(t *testing.T) {
+	// AppServiceClient needed for any Action that calls ensureLoggedIn.
```

Comment:
```suggestion
```
This comment is not relevant here; I was using it when I was passing in an empty inject client.

Reply:
oops -- good catch